test/index.js

```diff
@@ -123,6 +124,21 @@ test('File', t => {
   t.is(formData.get('foo'), foo)
 })
 
+test('Blob', t => {
+  const foo = new Blob([], {})
+  const formData = objectToFormData({
+    foo
+  })
+
+  t.true(formData.append.calledOnce)
+  t.deepEqual(formData.append.getCall(0).args, [
+    'foo',
+    foo
+  ])
+
+  t.true(formData.get('foo') instanceof File)
```

Comment:
Shouldn't this be `instanceof Blob`? So we don't check for an instance of a sub-prototype?

Reply:
When you append a Blob instance to FormData and call FormData.get() you'll get an instance of File instead of Blob.